### DISCUSSION TITLE
streamingcommunity.buzz

### DISCUSSION THREAD

**Discussion by Alessio979** [aclap-dev/video-downloadhelper#1188] [Help and Q&A]
### Your question

Hi, is there a way to copy all in one (audio/video) download link from this site?
Thanks

### Addon & CoApp version and details

version: 9.1.0.43
target: microsoft
channel: stable
lang: it
coapp: {"error":"Specified native messaging host not found.","found":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36 Edg/127.0.0.0
{
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "open_count_store": 4
}

**Comment by Mattmjs** [comment]
I got 5% of this https://streamingcommunity.buzz/watch/3877 before quitting as the speed is quite slow.
The file has both video and audio, so I don't know unless you have something specific that doesn't work.
What do you get on this example ?

**Reply by Alessio979** [reply]
What are you using to download links captured by video downloadhelper? via jdownloader it only downloads a file containing the playlist of the various streams. did you use pastylink+pastydownloader?
Thanks

**Reply by Mattmjs** [reply]
Oh I used VDH itself for the download, I don't use other software unless there is a specific case where it doesn't work.
Are you not using VDH to do the download ? It's works for your site but it is throttled.

**Reply by Mattmjs** [reply]
I might have tried jdownloader once and was not impressed with it that I uninstalled. The other software, never used it.

**Reply by Alessio979** [reply]
Do you use Coapp? i don't have it

**Reply by Mattmjs** [reply]
Yes the Coapp is needed if you are going to download it, but keep in mind on Edge there is a limit every 2 hours.
I use it in Firefox and no limit.

**Reply by Alessio979** [reply]
With PastyDownloader it works, but I confirm that it's very slow. It probably depends on the fact that it's a playlist.
Thanks
